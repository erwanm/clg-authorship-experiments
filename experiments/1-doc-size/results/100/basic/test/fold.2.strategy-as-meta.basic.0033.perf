0.282132	0.553200	0.510000
