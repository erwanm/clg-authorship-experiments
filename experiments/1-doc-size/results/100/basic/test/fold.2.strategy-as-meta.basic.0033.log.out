apply-config.sh: applying config '/tmp/100/process/selected-meta-configs/fold.2.strategy-as-meta.basic.0033' to cases '/home/moreaue/dump/expe1//results/100/basic/test/only-cases.tmp.Yaoqp9zF'; results written to '/home/moreaue/dump/expe1//results/100/basic/test/fold.2.strategy-as-meta.basic.0033'
apply-config.sh: done.
